File: comparison/results/haiku/codecraft/humaneval/HumanEval_36/typescript/solution.ts
function fizz_buzz(n: number): number {
  let count = 0;
  for (let i = 1; i < n; i++) {
    if (i % 11 === 0 || i % 13 === 0) {
      const str = i.toString();
      for (const char of str) {
        if (char === '7') {
          count++;
        }
      }
    }
  }
  return count;
}

File: comparison/results/haiku/codecraft/humaneval/HumanEval_36/java/solution.java
public static long fizzBuzz(long n) {
    long count = 0;
    for (long i = 1; i < n; i++) {
        if (i % 11 == 0 || i % 13 == 0) {
            String s = String.valueOf(i);
            for (char c : s.toCharArray()) {
                if (c == '7') {
                    count++;
                }
            }
        }
    }
    return count;
}
